File: C#/0125. Valid Palindrome.cs
public class Solution {
    public bool IsPalindrome(string s) {
        s=s.ToLower();
        IList<char> arr =new List<char>();
        string str="qwertyuiopasdfghjklzxcvbnm1234567890";
        foreach(char ss in s){
            if(str.Contains(ss)){
                arr.Add(ss);
            }
        }
        int left=0;
        int right=arr.Count-1;
        bool result=true;
        while(left<right){
            if(arr[left]!=arr[right]){
                result=false;
                break;
            }
            else{
                left++;
                right--;
            }
        }
        return result;
    }
}

File: 20190519python/0125. Valid Palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s=s.lower();
        rep=""
        for ss in s:
            if ss in "qwertyuiopasdfghjklzxcvbnm1234567890":
                rep+=ss
        return rep==rep[::-1]
